tests/test_project/test_project/viewtest/tests.py

```diff
@@ -118,6 +119,23 @@ def test_materialized_view(self):
         self.assertEqual(models.MaterializedRelatedViewWithIndex.objects.count(), 1,
             'Materialized view should have updated concurrently')
 
+    def test_signals(self):
+        synced_views = []
+        all_views_were_synced = [False]
+
+        @receiver(view_synced)
+        def on_view_synced(sender, has_changed, **kwargs):
+            synced_views.append(sender)
```

Comment:
Is it worth adding some information to the extra arguments here to make sure they're all correct?